docs/04-reference/wingsdk-spec.md

```diff
@@ -292,22 +292,63 @@ resource Bucket {
 
   /**
    * Get a file from the bucket.
+   *
+   * @throws Will throw if the file doesn't exist.
```

Comment:
It's worth noting, even though these methods specifically throw for functional reasons, **methods without `@throws` can still throw because they're networked APIs**, and pretty much every API call made on the internet is a best effort attempt. Just something to think about for the future